python/cuvs/pyproject.toml

```diff
@@ -32,7 +32,7 @@ license = { text = "Apache 2.0" }
 requires-python = ">=3.9"
 dependencies = [
     "cuda-python",
-    "numpy>=1.23,<2.0a0",
+    "numpy>=1.23,<3.0a0",
```

Comment:
Looks like we require NumPy here and the library does use it. However not seeing a dependency in the Conda recipe. Could you please add this same constraint there?

Reply:
Done, please have a look.  Will mark as ready for review as well.

Reply:
Looks good. Thanks! 🙏